apis/core/lib/domain/job/create.ts

```diff
@@ -74,9 +79,75 @@ export async function createPublishJob({
     publishedTo: metadata?.pointer.out(schema.workExample).term,
   })
 
+  const cubeId = isCsvProject(project)
+    ? organization.createIdentifier({
+      cubeIdentifier: project.cubeIdentifier,
+    })
+    : project.sourceCube
+
+  const dataset = await store.getResource<Dataset>(project.dataset.id)
+  const workExamples = prepareWorkExamples(cubeId, job, dataset)
+  workExamples.forEach(workExample => job.addWorkExample(workExample))
```

Comment:
Now adding the links to lindas and visualize when creating the job instead of after the job is finished. It's slightly less good on the UI side, but it allows us to re-use these "work examples" in the publish pipeline.